src/machine.adoc

```diff
@@ -2508,14 +2508,16 @@ available only to machine-mode.
 
 ==== Main Memory versus I/O versus Vacant Regions
 
-The most important characterization of a given memory address range is
-whether it holds regular main memory, or I/O devices, or is vacant.
+The most important characterization of a specific memory address range is
+whether it holds regular main memory or I/O devices, or if it is vacant.
 Regular main memory is required to have a number of properties,
-specified below, whereas I/O devices can have a much broader range of
-attributes. Memory regions that do not fit into regular main memory, for
-example, device scratchpad RAMs, are categorized as I/O regions. Vacant
-regions are also classified as I/O regions but with attributes
-specifying that no accesses are supported.
+which are specified in the following sections. However, I/O device regions can have a much broader  
+range of attributes. Address regions that do not fit into regular main memory are categorized as 
+I/O; for example, regions that contain device control registers. Finally, there are 
+other address regions that are neither memory nor I/O. These regions are called vacant
+regions. Vacant regions include attributes that specify access to these regions isn't 
+supported. Vacant regions can also be classified as I/O regions, but are never accessible. Any 
+attempt to access a vacant region causes an exception.
```

Comment:
Is it right to say "access-fault exception" here?

Reply:
if the exception is an access-fault exception, then we can be specific!

Reply:
Yes, it is correct to specify that a violation results in an access-fault exception - especially since this is explicitly stated early on in the PMA section (but is worth repeating here).